### PR TITLE
All properties from properties.json included in the package.json

### PR DESCRIPTION
**Pull request by bartoszglow**
## Description

AEM Site Template properties which are used by the Site Wizard are now moved from the `properties.json` and included in the `pakcage.json` to not duplicated some of them.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x]  I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x]  All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+  "description": "Basic AEM site template that serves as a clean starting point.",
   "license": "MIT License, Copyright 2020 Adobe Systems Incorporated",
   "scripts": {
     "build": "aem-site-template-builder",
     "deploy": "curl -u admin:admin -F 'file=@site-template.zip' http://localhost:4502/conf/global/site-templates.import.html"
   },
   "devDependencies": {
     "@adobe/aem-site-template-builder": "0.2.0"
+  },
+  "site_template": {
```
Comment (basecode) [inline package.json:13]: Looking [at the spec](https://docs.npmjs.com/cli/v6/configuring-npm/package-json) and seeing the usage of camel case like `devDependencies `, I would propose to rename to `siteTemplate` for consistency reasons.

**Review by basecode — APPROVED** [review]
One comment, otherwise good!

**Review by basecode — COMMENTED** [review]
just for the record :) missing task: update dependency to template builder pkg

File: package.json [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+    "createdBy": "Adobe",
+    "useCases": [],
+    "docsUrl": "",
+    "showcaseUrl": "",
+    "sourceRepositoryUrl": "https://github.com/adobe/aem-sites-template-basic"
```
Comment (jckautzmann) [inline package.json:18]: typo in the URL: sites -> site

**Comment by bartoszglow** [comment]
Waiting for #1 to be merged in first. Then we can release new patch version of AEM-site-template-builder and update this PR with proper npm dependency.

File: package.json [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
   "license": "MIT License, Copyright 2020 Adobe Systems Incorporated",
   "scripts": {
     "build": "aem-site-template-builder",
     "deploy": "curl -u admin:admin -F 'file=@site-template.zip' http://localhost:4502/conf/global/site-templates.import.html"
   },
   "devDependencies": {
     "@adobe/aem-site-template-builder": "0.2.0"
+  },
+  "siteTemplate": {
+    "createdBy": "Adobe",
```
Comment (basecode) [inline package.json:14]: `title` got lost.

**Review by basecode — CHANGES_REQUESTED** [review]

**Review by basecode — APPROVED** [review]

**Pull request merged**